frontend/components/notifications/NotificationsClientsTable.vue

```diff
@@ -0,0 +1,160 @@
+
+const colsVisible = computed(() => {
+  return {
+    footer: 1024,
+    threshold: width.value > 830,
+  }
+})
+
+const clientLink = computed(() => {
+  // TODO: implement client link when it's available from the API
+  // TODO: Test the endpoints
+  // return `/client/${data.clientId}`
+  return '/notifications'
+})
+</script>
+
+<template>
+  <div>
+    <BcTableControl
+      :title="$t('notifications.clients.title')"
+      :search-placeholder="$t('notifications.dashboards.search_placeholder')"
+      @set-search="setSearch"
+    >
+      <template #table>
+        <ClientOnly fallback-tag="span">
+          <BcTable
+            :data="clientsNotifications"
+            data-key="notification_id"
+            :cursor
+            :page-size
+            :selected-sort="query?.sort"
+            :loading="isLoading"
+            :add-spacer="true"
+            @set-cursor="setCursor"
+            @sort="onSort"
+            @set-page-size="setPageSize"
+          >
+            <Column
+              field="machine_name"
+              sortable
+              header-class="col-client-name"
+              body-class="col-client-name"
+              :header="$t('notifications.clients.col.client_name')"
+            >
+              <template #body="slotProps">
+                <pre>
+                  {{ slotProps.data.client_name }}
+                </pre>
+              </template>
+            </Column>
+            <Column
+              field="machine_name"
+              header-class="col-client-name"
+              body-class="col-client-name"
+              :header="$t('notifications.clients.col.version')"
+            >
+              <template #body="slotProps">
+                <BcLink
+                  :to="clientLink"
+                  class="link link-dashboard"
+                  target="_blank"
+                >
+                  {{ slotProps.data.version }}
+                </BcLink>
+              </template>
+            </Column>
+            <Column
+              field="timestamp"
+              sortable
+              header-class="col-age"
+              body-class="col-age"
+            >
+              <template #header>
+                <BcTableAgeHeader />
+              </template>
+              <template #body="slotProps">
+                <BcFormatTimePassed
+                  :value="slotProps.data.timestamp"
+                  type="go-timestamp"
+                />
+              </template>
+            </Column>
+            <template #empty>
+              <NotificationsDashboardsTableEmpty
+                v-if="!clientsNotifications?.data.length"
+                @open-dialog="$emit('openDialog')"
+              />
+            </template>
+            <!-- TODO: implement number of clients subscriptions -->
+            <template #bc-table-footer-right>
+              <template v-if="colsVisible">
+                {{ $t('notifications.clients.footer.subscriptions', { count: 3 }) }}
```

Comment:
plz also remove this `hardcoded` `count`

Reply:
will be implemented in BEDS-475